presto-spark-base/src/main/java/com/facebook/presto/spark/LocalBroadcastStorage.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark;
+
+import java.io.IOException;
+
+import static java.lang.String.format;
+import static java.nio.file.Files.createTempDirectory;
+
+public class LocalBroadcastStorage
+        implements BroadcastStorage
+{
+    @Override
+    public String getBroadcastDirectoryPath(String queryId)
+    {
+        String directoryPrefix = format("%s_%s", "local_broadcast", queryId);
+        try {
+            return createTempDirectory(directoryPrefix).toAbsolutePath().toString();
```

Comment:
I imagine this won't work on a cluster.

Reply:
Yes, this is local storage meant for testing. Distributed storage specific implementations will be added.